chromium/preloadlist/preloadlist.go

```diff
@@ -47,6 +47,31 @@ type PreloadList struct {
 	Entries []Entry `json:"entries"`
 }
 
+// PolicyType represents the policy under which the domain is a part of the preload list
+type PolicyType string 
+
+// Possible PolicyType values are as defined by 
+// https://source.chromium.org/chromium/chromium/src/+/main:net/http/transport_security_state_static.json
+const (
+	// test domains
+	Test PolicyType = "test"
+	// Google-owned sites
+	Google PolicyType = "google"
```

Comment:
Nit: You can omit the "PolicyType" type starting from the second one, so this looks like:
const (
	// test domains
	Test PolicyType = "test"
	// Google-owned sites
	Google                = "google"
...

Reply:
Done